version/1/4/de/code_of_conduct.md

```diff
@@ -0,0 +1,43 @@
+# Vereinbarung über Verhaltenskodex für Mitwirkende
+
+## Unsere Verpflichtung
+
+Im Interesse der Förderung eines offenen und einladenden Umfeldes wollen wir uns als Teilnehmer und Verantwortliche unseres Projektes verpflichten die Teilnahme an dem Projekt und unserer Gemeinschaft zu einer belästigungsfreien Erfahrung zu machen – unabhängig von Alter, Körpergröße, Behinderung, ethnischer Zuordnung, geschlechtlicher Identität und Ausdruck, Erfahrungsstufe, Nationalität, persönlicher Erscheinung, Rasse, Religion oder sexueller Identität oder Orientierung
+
+## Unsere Standards
+
+Beispiele für Verhaltensweisen, welche dazu beitragen ein positives Umfeld zu erzeugen beinhalten
+
+* Die Verwendung von einladenden und einbindenden Formulierungen
+* Abweichenden Ansichten und Erfahrungen wird Respekt entgegengebracht
+* Konstruktive Kritik wird höflich entgegengenommen
+* Fokussierung auf das, was das Beste ist für die Gemeinschaft
+* Verständnis zeigen gegenüber anderen Mitgliedern der Gemeinschaft
+
+Beispiele für nicht akzeptables Verhalten beinhalten:
+
+* Die Verwendung sexualisierter Sprache, Bilder oder Symbolik sowie unerwünschte Versuche sexueller Anbahnung
+* Nutzung der Anonymität des Internets für unbilliges Verhalten, beleidigende / abwertende Kommentare, persönliche oder politische Angriffe
+* Öffentliche oder private Belästigungen
+* Das Veröffentlichen von privaten Informationen Anderer, wie zum Beispiel physische oder elektronische Adressen, ohne deren ausdrückliche Erlaubnis
+* Anderes Verhalten, welches begründet als unangemessen betrachtet werden kann in einem professionellen Umfeld
+
+## Unsere Verantwortlichkeiten
+
+Die Projektverantwortlichen sind verantwortlich dafür die Standards für ein akzeptables Benehmen klarzustellen und es wird von ihnen erwartete, dass sie passende und faire berichtigende Maßnahmen ergreifen als Reaktion auf jegliches Vorkommen von inakzeptablen Verhalten.
```

Comment:
* Add a comma after "dafür"
* "erwartete" → "erwartet"
* I'd suggest "…, dass sie als Reaktion auf jegliches inakzeptables Verhalten passende und faire berichtigende Maßnahmen ergreifen." as a better ordering of the sentence